.github/workflows/ci.yml

```diff
@@ -0,0 +1,33 @@
+name: CI
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    branches: ["main"]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
+
+      - name: Build
+        run: GOOS=js GOARCH=wasm go build ./cmd/astro
```

Comment:
This tests that the build can still happen

Reply:
We're actually using [`tinygo`](https://tinygo.org/) to build but `GOARCH=wasm` might be easier for CI? Not sure if there's a setup action for `tinygo`

Reply:
Specifically `tinygo build -target=wasm -no-debug -o=astro.wasm ./cmd/astro `